blazor/data/getting-started.md

```diff
@@ -17,42 +17,41 @@ This section briefly explains about how to include [Blazor DataManager](https://
 
 ## Create a new Blazor App in Visual Studio
 
-You can create **Blazor Server App** or **Blazor WebAssembly App** using Visual Studio in one of the following ways,
+You can create a **Blazor Server App** or **Blazor WebAssembly App** using Visual Studio via [Microsoft Templates](https://learn.microsoft.com/en-us/aspnet/core/blazor/tooling?view=aspnetcore-7.0) or the [Syncfusion Blazor Extension](https://blazor.syncfusion.com/documentation/visual-studio-integration/template-studio).
 
-* [Create a Project using Microsoft Templates](https://learn.microsoft.com/en-us/aspnet/core/blazor/tooling?view=aspnetcore-7.0&pivots=windows)
+## Install Syncfusion Blazor Data and Theme NuGet in the App
```

Comment:
Do you need Theme nuget? since it is library, seems Theme is not needed

Reply:
Yes, it's needed because the DataManager component depends on another component (such as Data Grid, Dropdown, etc.). Therefore, a theme is required.